ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -1034,6 +1110,28 @@ def test_general_inner_product(*, data, test_flags, backend_fw, fn_name, on_devi
     )
 
 
+@handle_test(
+    fn_tree="functional.ivy.experimental.higher_order_moment",
+    data=_higher_order_moment_data(),
+)
+def test_higher_order_moment(*, data, test_flags, backend_fw, fn_name, on_device):
+    input_dtypes, x, order = data
+    if backend_fw == "paddle":
+        # to avoid large dimension results since paddle don't support them
+        order = min(order, 2)
```

Comment:
I'm not sure if I follow this- could you please explain why this is needed?

Reply:
the paddle backend does not support tensors with dimensions bigger than 6 (`len(tensor.shape) > 6`)  and since any addition in  `order `make an additional iteration in `higher_order_moment` function which add more dimensions to the returned tensor.

Reply:
Ah, makes sense. Thanks for explaining :+1: